Assets/Scripts/SS3D/Systems/Inventory/Items/Item.cs

```diff
@@ -272,11 +273,26 @@ public bool HasTrait(Trait trait)
         /// Modify the container of this item, can pass null to make this item not depending on any container.
         /// </summary>
         [Server]
-        public void SetContainer(AttachedContainer newContainer)
+        public virtual void SetContainer(AttachedContainer newContainer)
         {
-            if (_container == newContainer)
+            IEnumerable<Collider> colliders = GetComponents<Collider>().Concat(GetComponentsInChildren<Collider>());
+
+            if ((newContainer == null) != (_container == null))
             {
-                return;
+                if (newContainer == null)
```

Comment:
I wonder if this logic would not be better placed in the attached container class in handleitemadded and removed, the reason why I would do that is that maybe we want to enable disable colliders based on specific containers, but since I can't think of any current case where we'd want that for now, it's fine if it's stay there.

I'd just like this to be encapsulated in a private method (maybe called set colliders)